vmdb/app/models/container_definition.rb

```diff
@@ -1,5 +1,5 @@
 class ContainerDefinition < ActiveRecord::Base
   # :name, :image, :image_pull_policy, :memory, :cpu
   belongs_to :container_group
-  has_many :container_port_configs
+  has_many :container_port_configs, :dependent => :destroy
```

Comment:
If the container_definition "owns" the port configs, then yes, this is needed.  save_inventory handles updating existing records (including deletes), but if an object is *completely* eliminated, we rely on ActiveRecrd to cascade the deletes for us.